index.html

```diff
@@ -201,7 +195,7 @@ <h4 class="list-group-item-heading"><code>your-repo.git</code>/<code>branch.requ
 					</a>
 				</p>
 				<p class="text-right">
-					<copy>© <a href="//noelboss.com">Noël Bossart</a>. Made in Switzerland.</copy>
+					<small>© <a href="//noelboss.com">Noël Bossart</a>. Made in Switzerland.</small>
```

Comment:
Oops, I forgot the add `class="copy"` here. :disappointed: (vim complained that `<copy> wasn't a tag)